Meta/serenity_gdb.py

```diff
@@ -169,9 +189,7 @@ def to_string(self):
         if int(self.val["m_length"]) == 0:
             return '""'
         else:
-            characters = self.val["m_characters"]
-            str_type = characters.type.target().array(self.val["m_length"]).pointer()
-            return str(characters.cast(str_type).dereference())
+            return self.val["m_characters"].string(length=self.val["m_length"])
```

Comment:
I think that the "read the string as a string" part should be in its own commit

Reply:
You're right, looks cleaner this way. I reworded the commit message too, it was a bit awkwardly phrased.